### PR TITLE
Shipment model bug fix

### PR DESCRIPTION
**Pull request by zachholcomb**
# PR Description
Please include a summary of the change:
Fixes bug with the new shipment table.
Fixes bug in the user test.

:beetle: If it is a bug or issue fix, describe the issue that was fixed.
We needed to add the shipment model to one more file we forgot
Also updated the image to be the default in the user test.

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: code/api/src/setup/models.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ const models = {
   User: databaseConnection.import('../modules/user/model'),
   Product: databaseConnection.import('../modules/product/model'),
   Crate: databaseConnection.import('../modules/crate/model'),
-  Subscription: databaseConnection.import('../modules/subscription/model')
+  Subscription: databaseConnection.import('../modules/subscription/model'),
+  Shipment: databaseConnection.import('../modules/shipments/model')
```
Comment (zachholcomb) [inline code/api/src/setup/models.js:12]: We also needed to add this line to the models file which tells sequelize that the model file is a model

**Review by rachael-t — APPROVED** [review]
Thanks for getting this fixed so quickly!

File: code/api/tests/user.test.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe("user queries", () => {
     expect(response.body.data.user.role).toEqual('ADMIN')
     expect(response.body.data.user.id).toEqual(1)
     expect(response.body.data.user.email).toEqual('admin@crate.com')
-    expect(response.body.data.user.image).toEqual('https://images.unsplash.com/photo-1537815749002-de6a533c64db?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1145&q=80')
+    expect(response.body.data.user.image).toEqual('https://images.unsplash.com/photo-1537151608828-ea2b11777ee8?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=939&q=80')
```
Comment (zachholcomb) [inline code/api/tests/user.test.js:41]: Updated this back to the default image, it should now pass for everyone

File: code/api/src/setup/models.js (hunk 1 above)
Comment (rachael-t) [reply code/api/src/setup/models.js:12]: Thanks for clarifying that additional line!

File: code/api/tests/user.test.js (hunk 2 above)
Comment (rachael-t) [reply code/api/tests/user.test.js:41]: Perfect, thanks for taking a look at that!

**Review by Karlfunhouse — APPROVED** [review]

**Review by Karlfunhouse — APPROVED** [review]

File: code/api/src/setup/models.js (hunk 1 above)
Comment (infamouskeyduster) [reply code/api/src/setup/models.js:12]: I trust your judgement!

**Review by infamouskeyduster — APPROVED** [review]

File: code/api/src/setup/models.js (hunk 1 above)
Comment (atkinson-daniel) [inline code/api/src/setup/models.js:12]: Good catch on this!

**Review by atkinson-daniel — APPROVED** [review]

**Pull request merged**